torch/csrc/jit/register_special_ops.cpp

```diff
@@ -24,6 +26,35 @@ RegisterOperators reg({
         [](Stack& stack) {
           return 0;
         }),
+    Operator(
+        "aten::format(str self, *str args) -> Tensor",
+        [](Node* node) {
+          size_t num_inputs = node->inputs().size();
+          return [num_inputs](Stack& stack) {
+            auto format = peek(stack, 0, num_inputs).toStringRef();
+            auto args = last(stack, num_inputs - 1);
+
+            size_t base = 0;
+            std::string format_arg = "{}";
```

Comment:
Eh unfortunately the format syntax is very complicated, and we should give nicer error messages if someone uses `{5}`, `{some_arg_name}`, or even `{:<8.2f}`